modules/nw-throughput-troubleshoot.adoc

```diff
@@ -1,50 +1,39 @@
 // Module filename: nw-throughput-troubleshoot.adoc
 // Module included in the following assemblies:
-// * networking/configuring-routing.adoc
+// * networking/routes/route-configuration.adoc
 
+:_content-type: CONCEPT
 [id="nw-throughput-troubleshoot_{context}"]
-= Troubleshooting throughput issues
-Sometimes applications deployed through {product-title} can cause
-network throughput issues such as unusually high latency between
-specific services.
+= Throughput issue troubleshooting methods
 
-Use the following methods to analyze performance issues if pod logs do not
-reveal any cause of the problem:
+Sometimes applications deployed through {product-title} can cause network throughput issues such as unusually high latency between specific services.
```

Comment:
```suggestion
Sometimes applications deployed by using {product-title} can cause network throughput issues such as unusually high latency between specific services.
```
re: [ISG](https://www.ibm.com/docs/en/ibm-style?topic=word-usage#t) guidance on _through_.

Reply:
Forgot to add the note about the comma before _such as_